bitnami/grafana-operator add opt-in dashboardNamespaceSelector (#8204)
bitnami/charts: bitnami/grafana-operator/README.md

| `grafana.secrets`                                           | Extra secrets to mount into the grafana pod                                                   | `[]`                 |
| `grafana.jsonnetLibrarySelector`                            | Configuring the read for jsonnetLibraries to pull in.                                         | `{}`                 |
| `grafana.dashboardLabelSelectors`                           | This selects dashboards on the label.                                                         | `{}`                 |

> | `grafana.dashboardNamespaceSelector`                        | This selects dashboards only in the Namespaces that have the specified namespace label.       | `{}`                 |

Reviewer: ```suggestion
| `grafana.dashboardNamespaceSelector`                        | This selects dashboards only in the Namespaces that have the specified namespace label.       | `[]`                 |
```

> | `grafana.dashboardNamespaceSelector`                        | This selects dashboards only in the Namespaces that have the specified namespace label.       | `[]`                 |

Reviewer: ```suggestion
| `grafana.dashboardNamespaceSelector`                        | Watch for dashboards only in the Namespaces that have the specified namespace label       | `[]`                 |
```
